### DISCUSSION TITLE
is anysoftkeyboard still active?

### DISCUSSION THREAD

**Discussion by brookssw** [AnySoftKeyboard/AnySoftKeyboard#4037] [Q&A]
I'm interested in trying this, but on the play store it informs me that it's not compatible with my device, which is a Galaxy S22 Ultra. It looks like there is still active development, but only for older versions of android, is that right?

**Comment by xBaphomet666** [comment]
Same here. I've got a POCO F6 Pro running Android 14, and the Play Store page says that it "isn't available for [my] device because it was made for an older version of Android."

**Reply by sslaia** [reply]
I noticed the same today. I search for AnySoftKeyboard and it's not available

**Reply by Tronadis** [reply]
It is available on F-Droid.

**Comment by tsilvs** [comment]
Time for a fork then?

**Comment by timsterzel** [comment]
Now Anysoftkeyboard seem to be completly removed from playstore.

**Comment by flexjoly** [comment]
I can still find it, but not install it :-(

**Comment by Go-rom** [comment]
Hi

i have to admit that the development of Anysoft Keyboard is a mystery to me.

On the one hand, [everyone can see for themselves](https://github.com/AnySoftKeyboard/AnySoftKeyboard/commits/main/) that @menny spends a lot of time pushing this project forward, so it's very active, and on the other hand, it's getting harder and harder to get the latest version.

For those looking for recent versions, please check [this message](https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/3906#issuecomment-2131317602). It's not an easy way but it's working.

Also @Menny [has stated](https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/3979#discussioncomment-9020942) in last April that he would like to close [some issues](https://github.com/AnySoftKeyboard/AnySoftKeyboard/milestone/94) before releasing 1.12

**Reply by menny** [reply]
Yes... it's been very difficult to push a new version.. hard to fix bugs, new features, and ever new requirements by Google's Play Store.

Hopefully, I will be able to push something out shortly.

Sorry for the long delay.

**Reply by Ansarmughal99** [reply]
It has been a long time since

**Reply by Go-rom** [reply]
@Ansarmughal99 Still [three bugs](https://github.com/AnySoftKeyboard/AnySoftKeyboard/milestone/94) to fix

**Reply by Humpkes** [reply]
I had the same question. Looking for an answer lead me to this thread. I asked the IT-guys  at work to make AnySoftKeyboard available in our managed Play Store. They refused because the app hasn't been updated since 02.12.2021 in the Google Play Store and it conflicts with their setting for recent Android or something like that. So I checked GitHub for recent changes. I would say that there still are recent changes. Language packs are also updated. But not the core app. I would like to keep using it because it works like a charm. Hopefully they manage to push a new version. Would be great. Thanks in advance!
